### DISCUSSION TITLE
Loading local HRRR GRIB files

### DISCUSSION THREAD

**Discussion by gdiets** [blaylockbk/Herbie#112] [Q&A]
I'm a little inexperienced, so apologies if this is apparent... is there a way to use FastHerbie xarray to combine multiple local HRRR files? I don't want to trigger the download because (1) I already have the files and (2) I am looking at a large amount of data that is larger than the download was able to handle.  I remember reading language in the notes of an xarray definition that made it seem like there was functionality to load locally instead of downloading.

**Comment by gdiets** [comment]
I was able to achieve what I wanted pulling the following lines from the code:

![image](https://user-images.githubusercontent.com/31479593/194077924-c070dc80-888e-4993-b7db-fb3fa5812853.png)

**Reply by blaylockbk** [reply]
Glad you figured it out 😊
